server/selinux/server/pulp-streamer.te

```diff
@@ -76,3 +80,22 @@ ifdef(`distro_fedora23', `
     fs_rw_inherited_tmpfs_files(squid_t);
     fs_read_tmpfs_files(squid_t);
 ')
+
+
+##### Pulp stuff #####
+
+## Allow access to /var/run/pulp for EL 6
+ifdef(`distro_rhel6', `
+    allow streamer_t pulp_var_run_t:file manage_file_perms;
+    # dir access needed when the streamer searches for existing pid files on startup
+    allow streamer_t pulp_var_run_t:dir manage_dir_perms;
+    files_pid_filetrans(streamer_t, pulp_var_run_t, file)
+')
+
+
+##### Miscellaneous #####
+
+## Allow process to read localization information on EL 6
+ifdef(`distro_rhel6', `
+    miscfiles_read_localization(streamer_t)
```

Comment:
I did not determine why this was the case, but that is one possible explanation. It could also be due to something our init script itself is doing.